modules/ocf/manifests/auth.pp

```diff
@@ -132,11 +132,23 @@
     require => Augeas['sshd_config'],
   }
 
+  # Get all DNS names, FQDNs, and IPs for a host to include in SSH keys
+  $ssh_aliases = delete(concat(
+    suffix(delete(any2array($::dnsA), ''), ".${::domain}"),
+    $::dnsA,
+    suffix(delete(any2array($::dnsCname), ''), ".${::domain}"),
+    $::dnsCname,
+    $::fqdn,
+    $::ipHostNumber,
+    $::ipaddress6,
+  ), '')
+
   # Export SSH keys from every host, and use them to populate the global list
   # in /etc/ssh/ssh_known_hosts
```

Comment:
so uh... does this mean we can kill kerberos now?

Reply:
Hopefully? :P